openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KurentoHandler.java

```diff
@@ -346,9 +349,19 @@ JSONArray getTurnServers(final boolean test) {
 					turn.put("username", turnUser)
 						.put("credential", turnSecret);
 				}
-				final String fturnUrl = "turn:" + turnUrl;
-				turn.put("url", fturnUrl); // old-school
-				turn.put("urls", fturnUrl);
+
+				JSONArray urls = new JSONArray();
+				final String[] turnUrls = turnUrl.split(",");
+				for (int i = 0; i < turnUrls.length; i++) {
```

Comment:
_cosmetics_: `for (String url : turnUrls) {` will be more readable

will this code work if `turnUrl` is empty?

Reply:
kurento-utils.js will replace iceservers with freeice list if iceservers is empty:
https://github.com/Kurento/kurento-utils-js/blob/master/lib/WebRtcPeer.js#L260

Reply:
The question was about java code :)
I see there is a check in the beginning of the method :)